plutus-pab/src/Plutus/PAB/Run/Command.hs

```diff
@@ -36,6 +37,19 @@ data ConfigCommand =
     deriving stock (Show, Eq, Generic)
     deriving anyclass JSON.ToJSON
 
+
+-- | A single command to the PAB that spins up all the necessary services.
+allServices :: ConfigCommand
+allServices =
+  ForkCommands
+    [ StartMockNode
+    , ChainIndex
+    , Metadata
```

Comment:
FYI Metadata is going to be removed soon (https://github.com/input-output-hk/plutus/pull/3740)